wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -399,13 +396,10 @@ public Response getWorkspaces(
   })
   public WorkspaceDto update(
       @ApiParam("The workspace id") @PathParam("id") String id,
-      @ApiParam(value = "The workspace update", required = true) WorkspaceDto update)
+      @ApiParam(value = "The workspace update", required = true) String update)
```

Comment:
Since you change WorkspaceDto to String, is there a way to tell swagger expected model?